include/xtensor/xstrides.hpp

```diff
@@ -14,15 +14,11 @@
 #include <numeric>
 
 #include "xexception.hpp"
+#include "xlayout.hpp"
+#include "xtensor_forward.hpp"
 
```

Comment:
``xtensor_forward.hpp`` should not be included here since we do not manipulate tensor types in this file. If this is required by other files including ``xstrides.hpp``, then add the include in these files rather than in this one.